crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -171,7 +123,7 @@ pub trait PhaseItem: Sized + Send + Sync + 'static {
     fn draw_function(&self) -> DrawFunctionId;
 
     /// Sorts a slice of phase items into render order. Generally if the same type
-    /// implements [`BatchedPhaseItem`], this should use a stable sort like [`slice::sort_by_key`].
+    /// is batched this should use a stable sort like [`slice::sort_by_key`].
```

Comment:
Wording of "if the same type is batched" is awkward.

Reply:
I've improved the wording here, let me know if it sounds reasonable.

Reply:
Looks good now.